modules/apps/commerce/commerce-frontend-js/src/main/resources/META-INF/resources/components/mini_cart/CartItemsList.js

```diff
@@ -26,20 +26,37 @@ export default function CartItemsList({showPriceOnApplicationInfo = false}) {
 		cartState,
 		isUpdating,
 		labels,
+		replacementSKUList,
```

Comment:
Why capital letters for the Sku? Let's keep it consistent with other names, either change this or the other ones

Reply:
> https://github.com/liferay-commerce/liferay-portal/pull/3859#discussion_r1239795454 😄 
done, I kept the name used in the client

Reply:
Those are what the server expects, and I can't modify them. @alessiorendina